### PR TITLE
Avoid introducing `run` twice in the Rust book

### PR DESCRIPTION
**Pull request by BlueSpaceCanary**
As it stands, getting-started.md and guessing-game.md both introduce `run` as a new command. I switched it so that the 2nd refers back to the first introduction, rather than re-introducing the command.

(First ever FOSS PR, sorry if I screwed up anything obvious :) )

r? @steveklabnik

### CODE REVIEW

File: src/doc/book/getting-started.md [hunk 1]
```diff
@@ -504,6 +504,8 @@ $ cargo run
      Running `target/debug/hello_world`
 Hello, world!
 ```
+The `run` command comes in handy when you need to rapidly iterate on a
```
Comment (GuillaumeGomez) [inline src/doc/book/getting-started.md]: Missing an empty line before this one.

**Review by GuillaumeGomez — CHANGES_REQUESTED** [review]
All good except for the nit. Update, squash and then I r+.

File: src/doc/book/getting-started.md (hunk 1 above)
Comment (BlueSpaceCanary) [inline src/doc/book/getting-started.md]: Fixed!

**Comment by GuillaumeGomez** [comment]
Thanks!

@bors: r+ rollup

**Pull request merged**